ESPixelStick/src/platformDefinitions/GPIO_Defs_ESP32_Olimex_Gateway.hpp

```diff
@@ -0,0 +1,106 @@
+#pragma once
+/*
+* GPIO_Defs_ESP32_Olimex_Gateway.hpp - Output Management class
+*
+* Project: ESPixelStick - An ESP8266 / ESP32 and E1.31 based pixel driver
+* Copyright (c) 2021 Shelby Merrick
+* http://www.forkineye.com
+*
+*  This program is provided free for you to use in any way that you wish,
+*  subject to the laws and regulations where you are using it.  Due diligence
+*  is strongly suggested before using this code.  Please give credit where due.
+*
+*  The Author makes no warranty of any kind, express or implied, with regard
+*  to this program or the documentation contained in this document.  The
+*  Author shall not be liable in any event for incidental or consequential
+*  damages in connection with, or arising out of, the furnishing, performance
+*  or use of these programs.
+*
+*/
+
+/*
+ * Pinout for Olimex ESP32-GATEWAY
+ * https://www.olimex.com/Products/IoT/ESP32/ESP32-GATEWAY/open-source-hardware
+ */
+#define SUPPORT_ETHERNET
+
+//Output Manager
+#define SUPPORT_UART_OUTPUT
+#define DEFAULT_UART_1_GPIO     gpio_num_t::GPIO_NUM_4     // Supposed to be SD Card, but R10 not populated
+#define UART_LAST               OutputChannelId_UART_1
+
+
+#define SUPPORT_RMT_OUTPUT
+#define DEFAULT_RMT_0_GPIO      gpio_num_t::GPIO_NUM_12    // Tested working
+#define DEFAULT_RMT_1_GPIO      gpio_num_t::GPIO_NUM_13    // Tested working
+#define DEFAULT_RMT_2_GPIO      gpio_num_t::GPIO_NUM_16    // Tested working
+#define DEFAULT_RMT_3_GPIO      gpio_num_t::GPIO_NUM_32    // Tested working
+/*
+  Notes:
+  - Got exceptions and board resets when using either of GPIO_NUM_36 or GPIO_NUM_35
+    as an RMT output
+  - GPIOs 4, 12, 13 are for SD Card, but are not actually wired to the SD Card on
+    board rev G (resistors R10, R11, R9 are not populated on the board)
+*/ 
+// TODO: 39 is also available on the header, maybe also use 33 (LED output)
+#define RMT_LAST                OutputChannelId_RMT_4
+
+// #define SUPPORT_OutputType_WS2801    // requires a change in the html directory
```

Comment:
Output type definitions have been moved to the end of the file. Please keep them consistent with the existing file layout. This makes future maintenance easier. I suggest taking a current version from another ethernet capable platform like the GPIO_Defs_ESP32_ESP3DEUXQuattro_DMX and adjusting it to your needs.